### ISSUE TITLE
Does the training really need a full 100 epochs?

### ISSUE THREAD

**Issue by wangml71** [gaoyuezhou/dino_wm#31]
I noticed that both the hyperparameter list in the paper and the Hydra config in the code specify 100 epochs. Do we actually need to train for that many rounds? Also, at roughly what loss value can we consider the training to have converged?
